devops/providers/azure-devops/templates/infrastructure/azure-pipeline-cicd-compose.yml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  environments: []
+  configurationMatrix: []
+
+stages:
+
+- stage: PublishBuildArtifact
+  jobs:
+  - template: azure-pipeline-initial-artifact.yml
+
+- ${{ each environment in parameters.environments }}:
+  - stage: ${{ environment }}_Build
+    jobs:
+    - template: azure-pipeline-build-stage.yml
```

Comment:
Could we make these conditional and swap different build or release templates?

Reply:
Are you asking about configuring these on a per-pipeline, per-stage or per-template level?